terra/contracts/token-bridge/src/contract.rs

```diff
@@ -120,6 +123,11 @@ const CHAIN_ID: u16 = 3;
 
 const WRAPPED_ASSET_UPDATING: &str = "updating";
 
+pub enum TransferType<A> {
+    WithoutPayload,
+    WithPayload { payload: A },
+}
+
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> StdResult<Response> {
     let mut bucket = wrapped_asset_address(deps.storage);
```

Comment:
TODO: it's likely that by the time we roll out this patch to mainnet, this migration will have already gone through, so it should be removed.